UefiCpuPkg/Library/CpuPageTableLib/CpuPageTableMap.c

```diff
@@ -390,6 +390,17 @@ PageTableLibMapInLevel (
       if (RETURN_ERROR (Status)) {
         return Status;
       }
+
+      OneOfPagingEntry.Pnle.Uint64 = 0;
+      // MU_CHANGE Start - Populate base address bits for non present pages

```

Comment:
The MU_CHANGE section should move up a line.

Reply:
Updated in latest.